### ISSUE TITLE
Undefined variables

### ISSUE THREAD

**Issue by elliorf** [youcefouadjer9614/Motion-Gesture-Recognition#2]
Hello, 
I try to run the code and I get the following errors: 
"data_tensor" is not defined [Ln 49]
"label_tensor" is not defined [Ln 50]
"train_sampler" is not defined [Ln 67]
"test_sampler" is not defined [Ln 68]

Could you help please??

**Comment by youcefouadjer9614** [comment]
Hello, 
I have edited the code in a new branch.
Best regards

**Comment by youcefouadjer9614** [comment]
Hello, 
A good start would be to use more data, then perform train/test evaluation by tweaking hyperparameters (e.g. learning rate, opimizer...)
